apps/search/js/providers/suggestions.js

```diff
@@ -44,8 +52,6 @@
 
     name: 'Suggestions',
 
-    remote: true,
```

Comment:
Ok, sounds good. I definitely think that we should remove the ```remote``` attribute from other providers, but we can do that in a follow-up. Thanks for investigating the alternative approach of using the global object. I think we're on the same page now, and this can probably land whenever you're ready.